src/sys/getrandom.rs

```diff
@@ -0,0 +1,48 @@
+//! Wrapper around getrandom.
+
+use crate::Result;
+use libc;
+
+libc_enum! {
+    /// How the random bytes should be filled
+    #[repr(u32)]
+    #[non_exhaustive]
+    pub enum RandomMode{
+        /// If  this  bit is set, then random bytes are drawn from the
+        /// random source (i.e., the same source as the /dev/random device)
+        ///  instead of the urandom source.
```

Comment:
We should not copy directly from the man page, which may have license restrictions, but instead summarize.